crates/sncast/tests/integration/data_transformer.rs

```diff
@@ -0,0 +1,1022 @@
+use core::fmt;
+use itertools::Itertools;
+use primitive_types::U256;
+use shared::rpc::create_rpc_client;
+use sncast::helpers::data_transformer::transformer::transform;
+use starknet::core::types::{BlockId, BlockTag, ContractClass, Felt};
+use starknet::core::utils::get_selector_from_name;
+use starknet::providers::Provider;
+use std::ops::Not;
+use test_case::test_case;
+use tokio::sync::OnceCell;
+
+const RPC_ENDPOINT: &str = "http://188.34.188.184:7070/rpc/v0_7";
+
+// https://sepolia.starkscan.co/class/0x02a9b456118a86070a8c116c41b02e490f3dcc9db3cad945b4e9a7fd7cec9168#code
+const TEST_CLASS_HASH: Felt =
+    Felt::from_hex_unchecked("0x02a9b456118a86070a8c116c41b02e490f3dcc9db3cad945b4e9a7fd7cec9168");
+
+static CLASS: OnceCell<ContractClass> = OnceCell::const_new();
+
+// 2^128 + 3
+// const BIG_NUMBER: &str = "340282366920938463463374607431768211459";
+
+async fn init_class() -> ContractClass {
+    let client = create_rpc_client(RPC_ENDPOINT).unwrap();
+
+    client
+        .get_class(BlockId::Tag(BlockTag::Latest), TEST_CLASS_HASH)
+        .await
+        .unwrap()
+}
+
+trait Contains<T: fmt::Debug + Eq> {
+    fn assert_contains(&self, value: T);
+}
+
+impl Contains<&str> for anyhow::Error {
+    fn assert_contains(&self, value: &str) {
+        self.chain()
+            .any(|err| err.to_string().contains(value))
+            .not()
+            .then(|| panic!("{value:?}\nnot found in\n{self:#?}"));
+    }
+}
+
+#[tokio::test]
+async fn test_function_not_found() {
+    let simulated_cli_input = vec![String::from("'some_felt'")];
```

Comment:
Why not just use some proper input? Or is it valid?

Reply:
Whats wrong with this? @cptartur

Reply:
It's just an old name for a variable. Changed this to `input`.